### PR TITLE
MISUV-5429 - Fix tax year message on Charge summary page

### PR DESCRIPTION
**Pull request by simondrugan16**
added toStrings to necessary places on Charge Summary page

### CODE REVIEW

File: app/views/ChargeSummary.scala.html [hunk 1]
```diff
@@ -273,7 +273,7 @@ <h1 class="govuk-heading-xl">
         <div id="payment-link-@{taxYearTo}">
             <a class="govuk-button" role="button" data-module="govuk-button" href="@controllers.routes.PaymentController.paymentHandoff(documentDetail.remainingToPay.toPence, origin)">
                 @messages("paymentDue.payNow")
-                <span class="govuk-visually-hidden">@messages("paymentDue.pay-now-hidden", taxYearFrom, taxYearTo)</span>
+                <span class="govuk-visually-hidden">@messages("paymentDue.pay-now-hidden", taxYearFrom.toString, taxYearTo.toString)</span>
```
Comment (simondrugan16) [inline app/views/ChargeSummary.scala.html:276]: can't find this in frontend + is of class="govuk-visually-hidden" so is maybe hidden anyway. tests pass so should I keep the change or just remove it as it doesn't really matter?

File: test/views/ChargeSummaryViewSpec.scala [hunk 2]
```diff
@@ -138,6 +138,7 @@ class ChargeSummaryViewSpec extends ViewSpec with FeatureSwitching{
     def paymentOnAccountRequest(number: Int) = s"Payment on account $number of 2 reduced by taxpayer request"
 
     def class2NicTaxYear(year: Int) = messages("chargeSummary.nic2TaxYear", year - 1, year)
+    def class2NicTaxYearWithStringMessagesInput(year: Int) = messages("chargeSummary.nic2TaxYear", (year - 1).toString, year.toString)
```
Comment (avelx) [inline test/views/ChargeSummaryViewSpec.scala:141]: class2NicTaxYear seems to be redundant method

**Review by avelx — APPROVED** [review]
approved

File: app/views/ChargeSummary.scala.html (hunk 1 above)
Comment (avelx) [reply app/views/ChargeSummary.scala.html:276]: Double check relevant documentation for more details
https://frontend.design-system.service.gov.uk/sass-api-reference/#example-using-a-custom-ratio

**Pull request merged**